tests/unit/StandaloneOfficeAddressFiling.spec.ts

```diff
@@ -300,12 +307,13 @@ xdescribe('Standalone Office Address Filing - Part 1 - UI', () => {
   })
 })
 
-xdescribe('Standalone Office Address Filing - Part 2A - Resuming with FAS staff payment', () => {
+describe('Standalone Office Address Filing - Part 2A - Resuming with FAS staff payment', () => {
   beforeEach(() => {
     // init store
     store.state.businessId = 'CP0001191'
     store.state.entityIncNo = 'CP0001191'
     store.state.entityName = 'Legal Name - CP0001191'
+    store.state.entityType = 'CP'
```

Comment:
This omission really messed with me!!!
Sometimes, entity type was set from a previous test, and sometimes it was actually set to 'BEN' (which really screws with the code execution). Basically you can't guarantee what order tests run in, so if you need something set, set it!